src/common/plugins/plugins-config.js

```diff
@@ -12,7 +12,8 @@ function evaluatePluginsConfig(options: KalturaPlayerOptionsObject): void {
   if (options.plugins) {
     const dataModel: Object = {
       pVersion: __VERSION__,
-      pName: __NAME__
+      pName: __NAME__,
+      target: options.targetId
```

Comment:
target is too generic term, suggest to change to domRootElementId

Reply:
done